app/assets/javascripts/discourse/app/templates/invites/show.hbs

```diff
@@ -16,14 +16,26 @@
         <p>{{i18n "invites.invited_by"}}</p>
         <p>{{user-info user=invitedBy}}</p>
 
-        <p>
-          {{html-safe yourEmailMessage}}
-          {{#if externalAuthsEnabled}}
-            {{i18n "invites.social_login_available"}}
-          {{/if}}
-        </p>
+        {{#unless isInviteLink}}
+          <p>
+            {{html-safe yourEmailMessage}}
+            {{#if externalAuthsEnabled}}
+              {{i18n "invites.social_login_available"}}
+            {{/if}}
+          </p>
+        {{/unless}}
 
         <form>
+
+          {{#if isInviteLink}}
+            <div class="input email-input">
+              <label>{{i18n "user.email.title"}}</label>
+              {{input type="email" value=email id="new-account-email" name="email" autofocus="autofocus"}}
+              &nbsp;{{input-tip validation=emailValidation id="account-email-validation"}}
```

Comment:
not a big fan of &nbsp; could we add spacing through css ?

Reply:
Done.